backend/src/main/java/gov/cdc/usds/simplereport/service/OrganizationService.java

```diff
@@ -71,23 +79,23 @@ public Organization getCurrentOrganization() {
     public Organization getOrganization(String externalId) {
         Optional<Organization> found = _repo.findByExternalId(externalId);
         if (found.isEmpty()) {
-            throw new IllegalGraphqlArgumentException("Organization could not be found");
-        } else {
-            return found.get();
-        }
+            throw new IllegalGraphqlArgumentException("An organization with that external ID does not exist");
+        } 
+        
+        return found.get();
```

Comment:
Can replace all of this with

    return found.orElseThrow(()->new IllegalGraphqlArgumentException("An organization with that external ID does not exist"));